torch/utils/_sympy/value_ranges.py

```diff
@@ -16,6 +19,8 @@
 
 __all__ = ["ValueRanges", "ValueRangeAnalysis", "bound_sympy"]
 
+T = TypeVar('T', sympy.Expr, SympyBoolean)
```

Comment:
Shouldn't this be a private TypeVar? (leading underscore)

Reply:
Oh, I don't actually know! When should a TypeVar ever be public lol

Reply:
When someone might one use it downstream code or their own type annotations (ie. extending an abstract base class or something).

Reply:
That's so strange to me though, I always expected you would just redefine a TypeVar if you needed to do that haha

Reply:
Ughhh adding the leading underscore is so uglyyyyy